hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -18,6 +18,8 @@
 # new_cnt.
 LoopWarps = Dict[int, Dict[int, int]]
 
+SEC_WIPE_EN = 0
+SEC_WIPE_CNT = 96
```

Comment:
I think this is a duplicate of a value defined in `state.py`: can we drop this one?